### DISCUSSION TITLE
Create unique field for two tables in the prisma schema

### DISCUSSION THREAD

**Discussion by goceee** [prisma/prisma#8201] [Q&A]
I am trying to create my postgresql database with two models: User and Organisation and would like to have a field called username in both where it would be unique for both tables, for example if `username: gd123` exists in the User model the database should not be able to add it to the Organisation model. I would like to do this through the `prisma.schema` file if possible.

Any suggestion would be appreciated.

**Comment by ryands17** [comment]
@goceee 👋 
In this case, can a User belong to multiple organisations or would a single User be only in a single Organisation?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
